.gitignore

```diff
@@ -38,5 +38,6 @@ workflow
 .cache
 scripts/.srl
 privkey.pem
+app-autoscaler-release
```

Comment:
Why do we need this?

Reply:
I think `make deploy-autoscaler` creates that sub-directory - it should probably rather be in `build/`.